### DISCUSSION TITLE
PIC24f microcontroller question

### DISCUSSION THREAD

**Discussion by ltarmcool** [RudolphRiedel/FT800-FT813#53] [Q&A]
Hi Rudolph,
Your library works great with Arduino platform, I was able to use Arduino Uno/Micro/Nano328 to control this LCD NHD-4.3-480272FT-CTXL-T (**_https://newhavendisplay.com/content/specs/NHD-4.3-480272FT-CTXL-T.pdf_**). However, my project requires more features and I have to choose a different microcontroller (16 bit from PIC24f family), I wonder if your library will work with this PIC chip?
Thank you,

Lam

**Comment by RudolphRiedel** [comment]
I am very positive that it can work with pretty much any controller for which there is a C-compiler and that has enough memory. Bonus points if the controller has DMA support and enough SRAM for the 4kiB buffer.
I only never used PICs, had not never enough need for an alternative to use them.

Can you provide me with a code configurator example project with SPI and DMA that builds under MPLAB-X?
